src/collective/glossary/browser/static/resources.pt

```diff
@@ -0,0 +1,5 @@
+<link rel="stylesheet" tal:attributes="href string:${view/site_url}/++resource++collective.glossary/main.css" />
+<tal:if tal:condition="context/@@glossary_state">
```

Comment:
instead of a `tal` condition, we must handle this on the viewlet; it's easier and faster.

Reply:
I added an ajax call in the javascript to deal with it